bot/cogs/moderation/superstarify.py

```diff
@@ -1,34 +1,32 @@
+import json
 import logging
 import random
+from pathlib import Path
 
 from discord import Colour, Embed, Member
 from discord.errors import Forbidden
 from discord.ext.commands import Bot, Cog, Context, command
 
-from bot.cogs.moderation import Moderation
-from bot.cogs.modlog import ModLog
-from bot.cogs.superstarify.stars import get_nick
-from bot.constants import Icons, MODERATION_ROLES, POSITIVE_REPLIES
+from bot import constants
 from bot.converters import Duration
-from bot.decorators import with_role
-from bot.utils.moderation import post_infraction
+from bot.utils.checks import with_role_check
 from bot.utils.time import format_infraction
+from . import utils
+from .modlog import ModLog
 
 log = logging.getLogger(__name__)
 NICKNAME_POLICY_URL = "https://pythondiscord.com/pages/rules/#wiki-toc-nickname-policy"
```

Comment:
This URL has changed in the mean time by the removal of the `wiki-toc-` prefix:

```suggestion
NICKNAME_POLICY_URL = "https://pythondiscord.com/pages/rules/#nickname-policy"
```